content/nginx/deployment-guides/amazon-web-services/high-availability-keepalived.md

```diff
@@ -9,157 +9,220 @@ type:
+
 ## Step 3 – Install `keepalived`, `wget`, and the AWS CLI
 
-1. Install two packages from your OS vendor’s repository: the **keepalived** package and **wget**, which is used by the HA scripts.
+1. Install two packages from your OS vendor’s repository: the keepalived and wget. The HA scripts will use these.
 
    - On Ubuntu systems:
 
       ```shell
+
      sudo apt-get install keepalived wget
+
      ```
 
    - On CentOS systems:
 
       ```shell
+
      sudo yum install keepalived wget
+
      ```
 
-2. Follow the instructions in the [AWS documentation](https://docs.aws.amazon.com/cli/latest/userguide/installing.html) to install the AWS CLI.
+2. Follow the instructions in the [AWS documentation](https://docs.aws.amazon.com/cli/latest/userguide/installing.html) to install the AWS CLI.
 
 <span id="ha-aws_ha-scripts"></span><span id="ha-aws_step4"></span>
-## Step 4 – Download the HA Scripts
 
-The NGINX Plus HA solution uses two scripts, which are invoked by `keepalived`:
+## Step 4 – Download the HA scripts
+
+The NGINX Plus HA solution uses two scripts, which are invoked by `keepalived`:
+
+- <span style="white-space: nowrap; font-weight:bold;">nginx-ha-check</span> – Determines the health of NGINX Plus.
 
-- <span style="white-space: nowrap; font-weight:bold;">nginx-ha-check</span> – Determines the health of NGINX Plus.
 - <span style="white-space: nowrap; font-weight:bold;">nginx-ha-notify</span> – Moves the Elastic IP address when a state transition happens, for example when the backup instance becomes the primary.
 
 1. Create a directory for the scripts, if it doesn’t already exist.
 
    - On Ubuntu systems:
 
       ```shell
+
      sudo mkdir -p /usr/lib/keepalived
+
      ```
 
    - On CentOS systems:
 
       ```shell
+
      sudo mkdir -p /usr/libexec/keepalived
+
      ```
 
 2. Download the scripts from our [GitHub repository](https://github.com/nginxinc/aws-ha-elastic-ip) into the created directory.
 
 <span id="ha-aws_keepalived-configure"></span>
+
 ## Step 5 – Configure `keepalived` and the HA Scripts
 
 There are two configuration files for the HA solution:
 
-- **keepalived.conf** – The main configuration file for `keepalived`, slightly different for each NGINX Plus instance.
-- <span style="white-space: nowrap; font-weight:bold;">nginx-ha-notify</span> – The script you downloaded in [Step 4](#ha-aws_ha-scripts), with several user‑defined variables.
+- keepalived.conf – The main configuration file for `keepalived`, slightly different for each NGINX Plus instance.
+
+- <span style="white-space: nowrap; font-weight:bold;">nginx-ha-notify</span> – The script you downloaded in [Step 4](#ha-aws_ha-scripts), with several user‑defined variables.
 
 <span id="ha-aws_keepalived-conf-file"></span>
+
 ### Creating keepalived.conf
 
-In the **/etc/keepalived** folder create a file named **keepalived.conf** with the following content.
+In the /etc/keepalived folder create a file named keepalived.conf with the following content.
```

Comment:
We do want to emphasize the filename and directory. One common alternative: backquotes (or backticks), ref https://www.markdownguide.org/basic-syntax/#code

But for now, we'll retain the bolding:

```suggestion
In the **/etc/keepalived** folder create a file named **keepalived.conf** with the following content.
```